source/develop/release-management/features/network/networkfilterparameters.html.md

```diff
@@ -0,0 +1,282 @@
+associated to a network interface of the virtual machine.
+The new section provides the ability to add, edit and delete network filter parameters.
+If many network interface parameters are associated to the network interface, the section could be scrolled.
+Unfortunately there is no generic way to deduce the network filter parameters accepted by a network filter, so this section can not decide which parameter names are allowed.
+
+## Entity Description
+
+The top-level network filter is [associated to the vNIC profile][2]. For this reason it appears likely
+that the parameters for this filters, e.g. allowed networking protocols in a network,
+are associated to the vNIC profile, too.
+But the use case described in the [bug report][4] gives the VM's IP addresses as an example for parameters.
+These parameters are associated of a concrete network interface of the VM's instance.
+So there is a need for parameters on two places.
+
+In a first step, the parameters are implemented for VM's network interfaces,
+because it addresses the given use case in the [bug report][4].
+Furthermore there is the possibility to set parameters on VM's network interfaces,
+temporary solves the vNIC profile scenarios, too.
+The drawback of this approach is that in this scenarios more configuration needed,
+than it would be of the vNIC profiles could manage the filter parameters.
+
+The new entity ia called `VmNicFilterParameter`.
+It is associated to an network interface of an VM's network interface, and has the two attributes `name` and `value`.
+A `VmNicFilterParameter` is identified by it's `id`.
+All `VmNicFilterParameter` are persisted in the dedicated table `vm_interface_filter_parameters` in the database.
+
+### Adding a new network filter parameter
+
+To add a new network filter parameter, a reference to the VM's network interfaces,
+a valid name and a valid value are required.
+The name and the value has to satisfy at least the regular expressions of [libvirts XML schema][8],
+to ensure that VDSM will create valid XML for libvirt.
+There are [Reserved variables of libvirt's network filters][9] with known data types, which could be validated,
+but it is not planned to implement this.
+It is explicitly allowed to create multiple network filter parameters with the same name.
+
+On adding a new network filter parameter, a new row is created in the dedicated table in the database.
+
+### Updating a new network filter parameter
+
+The validation of the changed attributes are the same like for adding a new network filter parameter.
+
+Trough oVirt"s REST API it is possible to change the name and the value of the parameter.
+The entity allows to change the reference to the VM's network interface, but this is very unlikely to happen.
+The `id` of the entity must not be changed.
+
+On updating a network filter parameter, the row representing the network filter parameter is addressed by
+the attribute `id` and the other attributes in the row are updated.
```

Comment:
The description of "id" is quite obvious.
I don't have anything against it, but it does not add much value (but makes for more reading)